### PR TITLE
Debug a Node Js app running in Docker

### PR DESCRIPTION
**Pull request by geoffrey45**
I've written on debugging a Node JS app running in a Docker container. 

After reading the article the reader will be able to:
- Update Dockerfile and package.json to support debugging.
- Setup Nodemon for development environment
- Debug a Node Docker app in Chrome DevTools
- Debug a Node Docker app in VS Code.

The article has images and gifs in between. An *edited* hero image from Unsplash is also included.

### CODE REVIEW

**Comment by geoffrey45** [comment]
I've also updated my bio, added my socials and changed my avatar.

**Comment by hectorkambow** [comment]
Plagiarism check is done

**Comment by hectorkambow** [comment]
Awesome article - learned a lot - wanted to ask if we could change all the links to other EngEd article in markdown format - 
After that we can pass your article to tech review - before we publish. @geoffrey45

File: articles/debug-node-docker/index.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+ If you are new to Docker, I recommend you go through the following articles. They will give you better understanding on Docker.
+
+ 1. <a href="https://www.section.io/engineering-education/history-of-container-technology/" target="_blank">A Brief History of Container Technology</a>
+
+ 2. <a href="https://www.section.io/engineering-education/docker-concepts/" target="_blank">Understanding Docker Concepts</a>
+
+ 3. <a href="https://www.section.io/engineering-education/getting-started-with-docker/" target="_blank">Getting Started with Docker</a>
+
+ 4. <a href="https://www.section.io/engineering-education/building-a-nodejs-application-using-docker/" target="_blank">Building A Node.js Application Using Docker</a>
```
Comment (hectorkambow) [inline articles/debug-node-docker/index.md:33]: please add all these articles in .md format. 
[]()

File: articles/debug-node-docker/index.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+```bash
+$ docker run --rm -d  -p 3000:3000 -p  9229:9229  image:tag
+```
+
+The above command runs your container in detached mode (`-d`) and maps remote ports 3000 and 9229 to localhost ports 3000 and 9229 respectively.
+
+> Port 9229 is the default debug port for Node.js but you can change it if necessary.
+
+Finish up debugging your app in Chrome DevTools <a href="https://www.section.io/engineering-education/debug-node-devtools/#step-2----open-chrome-devtools" target="_blank">here</a>.
```
Comment (hectorkambow) [inline articles/debug-node-docker/index.md:128]: same with this article if we could put it in markdown format 🚀

**Comment by geoffrey45** [comment]
Hello @hectorkambow, I couldn't find a working syntax for opening links in new tab. So the links open in the same tab.

**Comment by hectorkambow** [comment]
Good afternoon 👋  @gwritz  - Happy Friday - wanted to ask if you could give this "Debug a Node Js app running in Docker" article a tech review before we publish whenever you have a moment. 
Thank you 🚀

**Review by gwritz — APPROVED** [review]

**Pull request merged**